react/FilterNavigator.js

```diff
@@ -103,6 +109,10 @@ const FilterNavigator = ({
     layout === LAYOUT_TYPES.mobile ||
     layout === LAYOUT_TYPES.phone
 
+  if(!Object.values(DRAWER_DIRECTION_MOBILE).includes(filtersDrawerDirectionMobile)) {
+    filtersDrawerDirectionMobile = DRAWER_DIRECTION_MOBILE.left
+  }
+
```

Comment:
It seems not to be a good idea to reassign properties, can you create a new variable for it?
Take a look on this suggestion
```ts
const drawerDirectionMobile = Object.values(DRAWER_DIRECTION_MOBILE).includes(filtersDrawerDirectionMobile) ? filtersDrawerDirectionMobile : DRAWER_DIRECTION_MOBILE.left
```
Lint my suggestion

Reply:
Since you asked that `drawerDirectionMobile` to be the name of the prop, i switched them. I hope it's ok, since this is an internal variable name.